client/src/ingredients/browser-compatibility-table/error-boundary.test.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import { BrowserCompatibilityErrorBoundary } from "./error-boundary.js";
+
+it("renders without crashing", () => {
+  const { container } = render(
+    <BrowserCompatibilityErrorBoundary>
+      <div />
+    </BrowserCompatibilityErrorBoundary>
+  );
+  expect(container).toBeDefined();
+});
+
+it("renders crashing mock component", () => {

```

Comment:
I actually haven't thought of a good way to write this scenario:

1. render `<BrowserCompatibilityTable />` happily
2. assert the happy condition
3. do something
4. `<BrowserCompatibilityTable />` crashes
5. assert error boundary is present

The closest I can get is by having a mock component (in this test case) 😅

Reply:
which I feel kind of make sense too to have this as a unit test, as this test purely test the behavior of Error Boundary without being affected by the many other components in BrowserCompatibilityTable